src/Classes/AcademicYear.php

```diff
@@ -21,23 +21,19 @@ class AcademicYear
      * @IS\Expose
      * @IS\Type("string")
      */
-    public $id;
+    public int $id;
 
     /**
      * @var string
```

Comment:
Can be dropped from comment, same with ID but I can't leave that comment for some reason.

Reply:
:heavy_check_mark: